content/developer/reference/frontend/framework_overview.rst

```diff
@@ -383,5 +386,77 @@ It is mostly interesting for testing purposes: all code using the browser object
 can be tested easily by mocking the relevant functions for the duration of the
 test.
 
+.. _javascript/debug_mode:
+
+Debug mode
+==========
+
+Odoo can sometimes operate in a special mode called the `debug` mode. It is used
+for two main purposes:
+
+- display additional information/fields for some particular screens,
+- provide some additional tools to help developer debug the Odoo interface.
+
+The `debug` mode is described by a string. An empty string means that the `debug`
+mode is not active. Otherwise, it is active.  If the string contains `assets` or
+`tests`, then the corresponding specific sub modes are activated (see below). Both
+modes can be active at the same time, for example with the string `assets,tests`.
+
+The `debug` mode current value can be read in the :ref:`environment<javascript/environment>`:
+`env.debug`.
+
+.. tip::
+
+    To show menus, fields or view elements only in debug mode, you should target
+    the group `base.group_no_one`:
+    
+    .. code-block:: xml
+
+        <field name="fname" groups="base.group_no_one"/>
+
+Toggling debug mode
+-------------------
+
+The most common way to activate the debug mode is to edit the url: just add
+`?debug=1` or `?debug=assets` before the `#` sign, then reload the page. This
+will cause the server to mark the session as being in debug mode.
+
+To deactivate the debug mode, one can then simply add `?debug=0` to explicitely
+notify the server that we no longer want it.
+
+An alternative way to toggle the debug mode is to use the command palette. First
+step is to open it (ctrl+k), then type `debug`: a command will show up to
+activate the debug mode.
```

Comment:
```suggestion
.. seealso::
   - :ref:`Activate the debug mode` <developer-mode>`
```

I think it's a bad idea to explain *how* to activate the debug mode in this technical doc. The page `applications/general/developer_mode.rst` already covers the *how*. This page should only cover the *what* (is the debug mode) and tips missing from the other page should be moved there.

The reason is that, not so long ago, there were 22 different guides on how to activate the developer mode in the user doc only. Fortunately, that number has gone down to only one and I believe it's for the best. While I understand that this page must cover the debug mode, it think it could be complementary to the other page ;)